[package] zeromq/4.3.2: file INSTALL cannot find

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **zeromq/4.3.2**
  * Operating System+version: **Linux Manjaro 19.0.2 Kyria**
  * Compiler+version: **clang version 9.0.1**
  * Conan version: **conan 1.22.3**
  * Python version: **Python 3.7.3**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Put your profile output here
```
Configuration for profile default:

[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=clang
compiler.version=9
compiler.libcxx=libstdc++11
build_type=Debug
[options]
[build_requires]
[env]
CC=/usr/bin/clang
CXX=/usr/bin/clang++
CONAN_CMAKE_GENERATOR=Ninja
### Steps to reproduce (Include if Applicable)

`conan install zeromq/4.3.2@ --build missing`


### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
Put your log output here
```

``` bash
98/100] Building CXX object source_subfolder/CMakeFiles/objects.dir/src/tipc_connecter.cpp.o
[99/100] Building CXX object source_subfolder/CMakeFiles/objects.dir/src/tipc_listener.cpp.o
[100/100] Linking CXX static library lib/libzmq.a
zeromq/4.3.2: Package '68ff68f2747785451ede57f58d04c7393f2e405a' built
zeromq/4.3.2: Build folder /home/jimmy/.conan/data/zeromq/4.3.2/_/_/build/68ff68f2747785451ede57f58d04c7393f2e405a
zeromq/4.3.2: Generated conaninfo.txt
zeromq/4.3.2: Generated conanbuildinfo.txt
zeromq/4.3.2: Generating the package
zeromq/4.3.2: Package folder /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a
zeromq/4.3.2: Calling package()
[0/1] Install the project...
-- Install configuration: "Debug"
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/lib/pkgconfig/libzmq.pc
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/lib/libzmq.a
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/include/zmq.h
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/include/zmq_utils.h
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/share/zmq/AUTHORS.txt
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/share/zmq/COPYING.txt
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/share/zmq/COPYING.LESSER.txt
-- Installing: /home/jimmy/.conan/data/zeromq/4.3.2/_/_/package/68ff68f2747785451ede57f58d04c7393f2e405a/share/zmq/NEWS.txt
CMake Error at source_subfolder/cmake_install.cmake:72 (file):
  file INSTALL cannot find
  "/home/jimmy/.conan/data/zeromq/4.3.2/_/_/build/68ff68f2747785451ede57f58d04c7393f2e405a/build_subfolder/source_subfolder/doc/zmq.html":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:42 (include)


FAILED: CMakeFiles/install.util
cd /home/jimmy/.conan/data/zeromq/4.3.2/_/_/build/68ff68f2747785451ede57f58d04c7393f2e405a/build_subfolder && /usr/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
ERROR: zeromq/4.3.2: Error in package() method, line 91
	cmake.install()
	ConanException: Error 1 while executing cmake --build '/home/jimmy/.conan/data/zeromq/4.3.2/_/_/build/68ff68f2747785451ede57f58d04c7393f2e405a/build_subfolder' '--target' 'install' '--' '-j8'
```

</details>

--- a/recipes/zeromq/all/conanfile.py
+++ b/recipes/zeromq/all/conanfile.py
@@ -59,7 +59,10 @@ class ZeroMQConan(ConanFile):
         self._cmake.definitions["WITH_PERF_TOOL"] = False
         self._cmake.definitions["BUILD_SHARED"] = self.options.shared
         self._cmake.definitions["BUILD_STATIC"] = not self.options.shared
+        self._cmake.definitions["BUILD_TESTS"] = False
         self._cmake.definitions["ENABLE_CPACK"] = False
+        self._cmake.definitions["WITH_DOCS"] = False
+        self._cmake.definitions["WITH_DOC"] = False
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 

